pkg/network/netlink/conntracker.go

```diff
@@ -281,30 +259,6 @@ func (ctr *realConntracker) run() {
 			}
 		}
 	}()
-
-	go func() {
-		for range ctr.compactTicker.C {
-			ctr.compact()
-		}
-	}()
-}
-
-func (ctr *realConntracker) compact() {
```

Comment:
If we're no longer periodically compacting this table and instead rely solely on the removal of expired keys to keep its size in check, should we be at all concerned about the potential size of this table?

Reply:
We do have a check for size in `register()`: https://github.com/DataDog/datadog-agent/blob/50430be0adfcb1c46aeadfc8d104b01743c3ca51/pkg/network/netlink/conntracker.go#L225